modules/hcp-dc-image-mirror.adoc

```diff
@@ -0,0 +1,98 @@
+
+Image mirroring is the process of fetching images from external registries, such as `registry.redhat.com` or `quay.io`, and storing them in your private registry.
+
+In the following procedures, the `oc-mirror` tool is used, which is a binary that uses the `ImageSetConfiguration` object. In the file, you can specify the following information:
+
+* The {product-title} versions to mirror. The versions are in `quay.io`.
+* The additional Operators to mirror. Select packages individually.
+* The extra images that you want to add to the repository.
+
+.Prerequisites
+
+Ensure that the registry server is running before you start the mirroring process.
+
+.Procedure
+
+To configure image mirroring, complete the following steps:
+
+. Ensure that your `${HOME}/.docker/config.json` file is updated with the registries that you are going to mirror from and with the private registry that you plan to push the images to.
+
+. By using the following example, create an `ImageSetConfiguration` object to use for mirroring. Replace values as needed to match your environment:
++
+[source,yaml]
+----
+apiVersion: mirror.openshift.io/v1alpha2
+kind: ImageSetConfiguration
+storageConfig:
+  registry:
+    imageURL: registry.<dns.base.domain.name>:5000/openshift/release/metadata:latest <1>
+mirror:
+  platform:
+    channels:
+    - name: candidate-4.17
+      minVersion: 4.x.y-x86_64  <2>
+      maxVersion: 4.x.y-x86_64 <2>
+      type: ocp
+    kubeVirtContainer: true <3>
+    graph: true
+  additionalImages:
+  - name: quay.io/karmab/origin-keepalived-ipfailover:latest
+  - name: quay.io/karmab/kubectl:latest
+  - name: quay.io/karmab/haproxy:latest
+  - name: quay.io/karmab/mdns-publisher:latest
+  - name: quay.io/karmab/origin-coredns:latest
+  - name: quay.io/karmab/curl:latest
+  - name: quay.io/karmab/kcli:latest
+  - name: quay.io/user-name/trbsht:latest
+  - name: quay.io/user-name/hypershift:BMSelfManage-v4.17
+  - name: registry.redhat.io/openshift4/ose-kube-rbac-proxy:v4.10
+  operators:
+  - catalog: registry.redhat.io/redhat/redhat-operator-index:v4.17
+    packages:
+    - name: lvms-operator
+    - name: local-storage-operator
+    - name: odf-csi-addons-operator
+    - name: odf-operator
+    - name: mcg-operator
+    - name: ocs-operator
+    - name: metallb-operator
+    - name: kubevirt-hyperconverged <4>
+----
++
+<1> Replace `<dns.base.domain.name>` with the DNS base domain name.
+<2> Replace `4.x.y` with the supported {product-title} version you want to use.
+<3> Set this optional flag to `true` if you want to also mirror the container disk image for the {op-system-first} boot image for the KubeVirt provider. This flag is available with oc-mirror v2 only.
+<4> For deployments that use the KubeVirt provider, include this line.
+
+. Start the mirroring process by entering the following command:
++
+[source,terminal]
+----
+$ oc-mirror --v2 --source-skip-tls --config imagesetconfig.yaml docker://${REGISTRY}
```

Comment:
`--source-skip-tls` is not available anymore in `oc-mirror v2` and our public registries are using globally trusted CA so please remove it.